cmd/root.go

```diff
@@ -180,6 +180,7 @@ func rootCmd(o *options.Options) error {
 		return fmt.Errorf("failed to format results: %w", resultsErr)
 	}
 
+	printCheckResults(enabledChecks)
```

Comment:
This should stay under the `if o.Format == options.FormatDefault` block, or we'll start printing it for `--format json`

```
go run main.go --repo=github.com/ossf-tests/scorecard-check-branch-protection-e2e \
--probes=fuzzed --format json
{"date":"2024-11-05T08:39:37-08:00","repo":{"name":"github.com/ossf-tests/scorecard-check-branch-protection-e2e","commit":"12ae42962014ee9aeb01d991ee2cd799ad6de659"},"scorecard":{"version":"devel","commit":"unknown"},"score":-1.0,"checks":null,"metadata":null}
Finished [Token-Permissions]
Finished [SAST]
Finished [Dangerous-Workflow]
Finished [Contributors]
Finished [Code-Review]
Finished [Binary-Artifacts]
Finished [Dependency-Update-Tool]
Finished [License]
Finished [Pinned-Dependencies]
Finished [Signed-Releases]
Finished [Security-Policy]
Finished [Fuzzing]
Finished [Vulnerabilities]
Finished [CII-Best-Practices]
Finished [Branch-Protection]
Finished [CI-Tests]
Finished [Packaging]
Finished [Maintained]

RESULTS
-------
```

Reply:
```suggestion
```